cmd/kubelet/app/options/options.go

```diff
@@ -55,6 +55,7 @@ const defaultRootDir = "/var/lib/kubelet"
 type KubeletFlags struct {
 	KubeConfig          string
 	BootstrapKubeconfig string
+	VersionJSON         bool
```

Comment:
These and the following related change, probably shouldn't be included in that PR.